content/docs/reference/pkg/python/pulumi_random/_index.md

```diff
@@ -368,6 +369,78 @@ a format of their choosing before sending those properties to the Pulumi engine.
 
 </dd></dl>
 
+<dl class="class">
```

Comment:
Hmm, Paul just re-generated the docs for this in #1531. Are you sure you have the latest changes from that repo and the `0.5.8` release branch checked out?

Reply:
This was due to the script for generating the Python API docs was installing pre-release packages. I have removed the flag in the `pip install` command that did that.